.changeset/smart-ants-decide.md

```diff
@@ -0,0 +1,5 @@
+---
+"@remix-run/router": patch
+---
+
+Support basename and relative routing in loader/action redirects
```

Comment:
Did this as a patch/bug fix (as opposed to a minor/new feature) since redirects probably should have supported `basename` and relative routing from the get go.  This would be new behavior for remix though when we bring it over - but I would argue non-breaking